packages/protocol/contracts/layer1/preconf/libs/LibPreconfConstants.sol

```diff
@@ -9,6 +9,7 @@ library LibPreconfConstants {
     uint256 internal constant ETHEREUM_MAINNET_BEACON_GENESIS = 1_606_824_023;
     uint256 internal constant ETHEREUM_HOLESKY_BEACON_GENESIS = 1_695_902_100;
     uint256 internal constant ETHEREUM_HELDER_BEACON_GENESIS = 1_718_967_660;
+    uint256 internal constant ETHEREUM_HOODI_BEACON_GENESIS = 0; // TODO: check this value
```

Comment:
@smartprogrammer93 Do you know if this value is correct? I didn't manage to find the info.

Reply:
Should be: 1742212800